reflex/__init__.py

```diff
@@ -304,7 +304,7 @@
         "window_alert",
     ],
     "middleware": ["middleware", "Middleware"],
-    "model": ["session", "Model"],
+    "model": ["session", "sqla_session", "Model"],
```

Comment:
I think this shouldn't be in the main `rx.`namespace as it's more of a special case. For people who need it, they can do the full import as `rx.model.sqla_session`

Reply:
Thanks for your review, done!